File: src/test/java/com/baulsupp/oksocial/completion/UrlCompleterMain.java
package com.baulsupp.oksocial.completion;

import com.baulsupp.oksocial.Main;

import static com.google.common.collect.Lists.newArrayList;

public class UrlCompleterMain {
  public static int main(String[] args) throws Exception {
    Main main = new Main();
    main.initialise();

    main.arguments =
        newArrayList(args.length > 0 ? args[0] : "https://graph.facebook.com/BooneStudio");
    return main.run();
  }
}

File: src/test/kotlin/com/baulsupp/oksocial/completion/UrlCompleterMain.kt
package com.baulsupp.oksocial.completion

import com.baulsupp.oksocial.Main

object UrlCompleterMain {
  fun main(args: Array<String>): Int {
    val main = Main()
    main.initialise()

    main.arguments = mutableListOf(
        if (args.isNotEmpty()) args[0] else "https://graph.facebook.com/BooneStudio")
    return main.run()
  }
}
